anda/misc/tracy/tracy.spec

```
@@ -0,0 +1,65 @@
+Name:			tracy
+Version:		0.10
+Release:		1%?dist
+Summary:		A real time, nanosecond resolution, remote telemetry, hybrid frame and sampling profiler for games and other applications.
+License:		BSD-3-Clause
+URL:			https://github.com/wolfpld/tracy
+Source0:		https://github.com/wolfpld/tracy/archive/refs/tags/v%version.tar.gz
+BuildRequires:  cmake meson gcc libxkbcommon dbus-devel libglvnd glfw-devel freetype-devel pkgconfig(capstone) pkgconfig(libunwind) pkgconfig(libdebuginfod) pkgconfig(tbb)
+Patch:          https://github.com/wolfpld/tracy/commit/1a971d867d6fa5bf6dc57d705dcbbc6020031e7a.patch
+
+%description
+%A real time, nanosecond resolution, remote telemetry, hybrid frame and sampling profiler for games and other applications.
+
+%prep
+%autosetup -n tracy-0.10
```

```suggestion
%autosetup
```

---

```suggestion
%autosetup -n tracy-0.10 -p1
```

If you look at the raw .patch file, the filenames specified in it are under the "fake directories" `a` and `b`. `-p1` tells %autosetup to strip that first directory from the path

---

Got it, thanks!